content/blog/grpc-js-1.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "@gprc/grpc-js 1.0 release announcement"
+date: 2020-04-20
+author:
+  name: Michael Lumish
+  link: https://github.com/murgatroid99
+---
+
+Today we are excited to announce the release of version 1.0 of [@grpc/grpc-js](https://www.npmjs.com/package/@grpc/grpc-js). This library is a reimplementation of the original [grpc](https://www.npmjs.com/package/grpc) library for Node.js, but purely in TypeScript, no longer depending on native C++ code that requires complex compilation and installation procedures.
```

Comment:
Nit: line-wrap all of the long lines. This is useful for a number of reasons, not the least of which is that it makes it easier to attach comments to specific phrases.

"Today ...":
- I would drop "Today" as that designation will only be valid for today :).

"This library is ... installation procedures":
- How about something like

  > The gRPC-JS library is a pure-TypeScript reimplementation of the original Node gRPC library ([grpc][]).

  [grpc]: https://www.npmjs.com/package/grpc

  I'd consider dropping the "no longer depending ... complex compilation and installation procedures", because if I look at the [Node Quickstart](https://grpc.io/docs/quickstart/node), I don't see any "complex compilation and installation procedures".

Actually, if you combine these, your first paragraph could read:

> We're excited to announce release 1.0 of gRPC-JS ([@grpc/grpc-js][]), a pure-TypeScript reimplementation of the original Node gRPC library, [grpc][].

  [@grpc/grpc-js]: https://www.npmjs.com/package/@grpc/grpc-js

Reply:
We never really said it in the quickstart because that was really our only option at the time. But this is the complain that people gave us over and over again.

Reply:
More specifically, if everything goes smoothly installation is simple, but it is error-prone and resolving the errors can be difficult and complicated. Still, I don't necessarily think it's worth bringing up problems that most users don't encounter, so I did remove that part.